openghg/store/_oneD_timeseries.py

```diff
@@ -0,0 +1,339 @@
+    #     if "time" in oned_data.coords:
+    #         oned_data = update_zero_dim(oned_data, dim="time")
+
+    #     # Currently ACRG boundary conditions are split by month or year
+    #     oneD_time = oned_data["time"]
+
+    #     start_date, end_date, period_str = infer_date_range(
+    #         oneD_time, filepath=filepath, period=period, continuous=continuous
+    #     )
+
+    #     # Checking against expected format for boundary conditions
+    #     OneDTimeseries.validate_data(oned_data)
+    #     data_type = "oned_timeseries"
+
+    #     metadata["start_date"] = str(start_date)
+    #     metadata["end_date"] = str(end_date)
+    #     metadata["data_type"] = data_type
+
+    #     metadata["input_filename"] = filepath.name
+
+    #     metadata["time_period"] = period_str
+
+    #     key = "_".join((species, domain))
+
+    #     oneD_data: DefaultDict[str, Dict[str, Union[Dict, Dataset]]] = defaultdict(dict)
+    #     oneD_data[key]["data"] = oned_data
+    #     oneD_data[key]["metadata"] = metadata
+
+    #     required_keys = ("species", "domain")
+
+    #     # This performs the lookup and assignment of data to new or
+    #     # exisiting Datasources
+    #     data_type = "oned_timeseries"
+    #     datasource_uuids = self.assign_data(
+    #         data=oneD_data,
+    #         if_exists=if_exists,
+    #         new_version=new_version,
+    #         data_type=data_type,
+    #         required_keys=required_keys,
+    #         compressor=compressor,
+    #         filters=filters,
+    #     )
+
+    #     # Record the file hash in case we see this file again
+    #     self._file_hashes[file_hash] = filepath.name
+
+    #     return datasource_uuids
+
+    @staticmethod
+    def validate_data(data: Dataset) -> None:
+        """
+        Validate input data against BoundaryConditions schema - definition from
+        BoundaryConditions.schema() method.
+
+        Args:
+            data : xarray Dataset in expected format
+
+        Returns:
+            None
+
+            Raises a ValueError with details if the input data does not adhere
+            to the BoundaryConditions schema.
+        """
+        data_schema = OneDTimeseries.schema()
+        data_schema.validate_data(data)
+
+    @staticmethod
+    def schema() -> DataSchema:
+        """
+        Define schema for one dimensional timeseries(OneDTimeseries) Dataset.
+
+        Includes observation for each time of the defined domain:
+            - "Obs"
+                - expected dimensions: ("time")
+
+        Expected data types for all variables and coordinates also included.
+
+        Returns:
+            DataSchema : Contains schema for OneDTimeseries.
+        """
+        from openghg.store import DataSchema
+
+        data_vars: Dict[str, Tuple[str, ...]] = {"oneD": ("time",)}
+        dtypes = {
+            "time": np.datetime64,
+            "oneD": np.floating,
```

Comment:
Would likely want to update e.g. something like "flux_timeseries"

Reply:
updated to flux as we discussed in the meeting to be consistent.

Reply:
@brendan-m-murphy - can you recall the discussion on this? For the 1D flux timeseries with (time,) dimension would we want this to have the same variable naming as for the 3D flux data with (time, lat, lon). We can do so but this could potentially cause confusion.

Reply:
Waiting for this, Apart from that I believe everything is resolved.

Reply:
Let's update the variable name to `flux_timeseries` (1D) for now to keep this distinct from `flux` (3D).